build.ps1

```diff
@@ -147,16 +147,36 @@ function RunTests {
         Write-Output "no test assemblies found in $testAssemblyPath"
     }
 
+    Write-Output "Tests Assemblies List"
+    Write-Output $testAssemblies
+    Write-Output "End Tests Assemblies List"
+
     $testAssemblies | ForEach-Object {
         Write-Output "Executing: dotnet test $($_)"
 
         $fileName = Split-Path -Path  $($_) -Leaf
         $fileNameNoExt = $fileName.subString(0, $fileName.length-4)
 
         Invoke-Execute {
-            dotnet test $_ `
-                --logger "trx;LogFileName=$testResults/$fileNameNoExt.trx" `
-                --nologo
+            if ($Filter.Equals("*.Tests.Unit")){
+                #Execution with coverage
+                # Threshold need to be defined
+                coverlet $($_) `
+                --target dotnet --targetargs "test $($_)" `
+                --threshold 50 `
```

Comment:
What is the lowest number that all unit test projects currently meet? Whatever that is, we should use it as the default.

Suggestion: create a constant with this number at the top of the file, so that it is easy to find the number and update it in the future.

Reply:
Current value is 58.53, I set that value as a variable.